react-app/package.json

```diff
@@ -90,10 +90,15 @@
     "typescript": "^5.1.3"
   },
   "peerDependencies": {
-    "@mui/icons-material": "^5.11.16",
+    "@emotion/react": "^11.11.4",
+    "@emotion/styled": "^11.11.5",
+    "@mui/icons-material": "^5.13.7",
     "@mui/material": "^5.13.5",
     "@mui/system": "^5.13.5",
     "react": "^17 || ^18",
     "react-dom": "^17 || ^18"
+  },
+  "volta": {
+    "node": "18.17.0"
```

Comment:
Any reason to not use latest 18 version?
You can resolve :)

Reply:
Not really, I picked an example I had at hand...
Updated to 18.20.3 (LTS)